src/piece_picker.cpp

```diff
@@ -413,7 +413,7 @@ namespace libtorrent
 		for (int i = 0; i < num_pieces; ++i)
 		{
 			int h = have[i];
```

Comment:
would it make more sense to widen ``h`` here instead of truncating the result from ``count()``?